src/vm/runners/builtin_runner.rs

```diff
@@ -92,24 +86,17 @@ impl RangeCheckBuiltinRunner {
 }
 impl BuiltinRunner for RangeCheckBuiltinRunner {
     fn initialize_segments(&mut self, segments: &mut MemorySegmentManager, memory: &mut Memory) {
-        self.base = Some(segments.add(memory, None))
+        self.base = segments.add(memory, None).segment_index
     }
-    fn initial_stack(&self) -> Result<Vec<MaybeRelocatable>, RunnerError> {
-        if self.included {
-            if let Some(builtin_base) = &self.base {
-                Ok(vec![MaybeRelocatable::RelocatableValue(
-                    builtin_base.clone(),
-                )])
-            } else {
-                Err(RunnerError::UninitializedBase)
-            }
-        } else {
-            Ok(Vec::new())
-        }
+
+    fn initial_stack(&self) -> Vec<MaybeRelocatable> {
+        vec![MaybeRelocatable::RelocatableValue(Relocatable::from((
+            self.base, 0,
```

Comment:
Can't `MaybeRelocatable::from(..., ...)` be used here?

Reply:
Sure, good call!